pkg/downstream/suggest.go

```diff
@@ -121,3 +122,58 @@ func iteratePullRequestCommits(ctx context.Context, client *github.Client, org,
 			return client.PullRequests.ListCommits(ctx, org, repo, prNum, o)
 		})
 }
+
+func hasCommit(git utils.GitHelper, req *SuggestRequest, found []string, c *github.Commit) (bool, error) {
+	has_commit := false
+	for _, commit := range found {
+		if len(commit) == 0 {
```

Comment:
If we have an empty line, should we just ignore it and keep inspecting the others instead of giving up?

Reply:
Given the way we use `hasCommit`, `commit` should only be empty when reaching the last element of `found`
However I agree on making the function itself more robust.